### PR TITLE
US 018-UI-SCE: UI Show History button operation / GET Endpoint HISTORY Integration with UI

### PR DESCRIPTION
**Pull request by moselGlez**
Added BE endpoint Get History by Logged USER.

### CODE REVIEW

File: src/config/passport.js [hunk 1]
```diff
@@ -30,20 +30,20 @@ passport.use(
         email: email,
         password: (await encrypt(password)).content,
       };
-      console.debug(request);
+      //console.debug(request);
```
Comment (at-internship) [inline src/config/passport.js]: This change is not necessary

File: src/config/passport.js [hunk 2]
```diff
@@ -30,20 +30,20 @@ passport.use(
...
 
       try {
         // Validate user
         const userAuth = await sceServiceAPI.login(request);
-        console.debug(userAuth);
+        //console.debug(userAuth);
```
Comment (at-internship) [inline src/config/passport.js]: This change is not necessary

File: src/config/passport.js [hunk 3]
```diff
@@ -30,20 +30,20 @@ passport.use(
...
 
         if (!userAuth && !userAuth.data.id) {
           console.error("Not User found: ", email);
           return done(null, false, { message: "Not User found." });
         } else {
           // Get User details
           const user = await sceServiceAPI.getUserById(userAuth.data.id);
-          console.debug(user);
+          //console.debug(user);
```
Comment (at-internship) [inline src/config/passport.js]: This change is not necessary

File: src/routes/at-sce.routes.js [hunk 4]
```diff
@@ -14,10 +14,10 @@ const router = express.Router();
 const path = require("path");
 
 // Helpers
-const { isAuthenticated } = require("../helpers/auth.helper");
+const { isAuthenticated} = require("../helpers/auth.helper");
```
Comment (at-internship) [inline src/routes/at-sce.routes.js:17]: This change is not necessary

File: src/routes/at-sce.routes.js [hunk 5]
```diff
@@ -14,10 +14,10 @@ const router = express.Router();
...
 
 // SCE Controller
-const { renderSigninForm, signin, signout, calculator, addHistory } = require("../controllers/at-sce.controller");
+const { renderSigninForm, signin, signout, calculator, addHistory} = require("../controllers/at-sce.controller");
```
Comment (at-internship) [inline src/routes/at-sce.routes.js:20]: This change is not necessary

**Review by at-internship — APPROVED** [review]
Approved

**Review by joseDnim — APPROVED** [review]

**Review by irvingesp2190 — APPROVED** [review]

**Pull request merged**